src/components/Workshops/helpers.ts

```diff
@@ -0,0 +1,22 @@
+import dayjs from "dayjs";
+
+import { Workshop, WorkshopSortType } from "../../types";
+
+export const sortEvents = (
+  events: Workshop[],
+  sortType: WorkshopSortType,
+  isOld: boolean
+) => {
+  if (sortType === WorkshopSortType.ByPrice) {
+    return events.sort((a, b) => a.price - b.price);
+  }
+  if (isOld) {
+    return events
+      .sort((a, b) => dayjs(a.startDate).unix() - dayjs(b.startDate).unix())
+      .reverse();
```

Comment:
I'd just switch the sort direction and remove reverse ()
```suggestion
      .sort((a, b) =>  dayjs(b.startDate).unix() - dayjs(a.startDate).unix())
```